src/data/Const.ts

```diff
@@ -4,3 +4,13 @@ export const DataChannelReadyState = {
     CLOSING: 'closing',
     CLOSED: 'closed',
 }
+
+export const PeerConnectionState = {
+    New: 'new',
```

Comment:
Should we make the constants all uppercase

Reply:
I am not a fan of uppercase constants.